app/views/decidim/action_delegator/admin/delegations/index.html.erb

```diff
@@ -1 +1,31 @@
-index
+<div class="card with-overflow" id='user-groups'>
+  <div class="card-divider">
+    <h2 class="card-title"><%= t ".title" %></h2>
+  </div>
+  <div class="card-section">
+    <div class="table-scroll">
+      <table class="table-list">
+        <thead>
+          <tr>
+            <th><%= t(".grantee") %></th>
+            <th><%= t(".granter") %></th>
+            <th><%= t(".created_at") %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% delegations.each do |delegation| %>
+            <tr data-delegation-id="<%= delegation.id %>">
+              <td><%= delegation.granter.name %></td>
+              <td><%= delegation.grantee.name %></td>
+              <td><%= l delegation.created_at, format: :short %></td>
+
+              <td class="table-list__actions">
+                <%= link_to t(".delete"), Decidim::ActionDelegator::AdminEngine.routes.url_helpers.delegation_path(delegation), method: :delete %>
```

Comment:
I guess we need to include the url_helpers in the engine file or somewhere else so we can use `delegation_path(delegation)` instead. Any idea @microstudi ?

Reply:
that why we create a global module, it create a global object to hold those routes. in this case `decidim_admin_action_delegator.delegation_path(delegation)`